scipy/stats/tests/test_distributions.py

```diff
@@ -2238,6 +2238,60 @@ def test_logcdf_logsf(self):
         logsf = stats.invgauss.logsf(110, 1.05)
         assert_allclose(logsf, -56.1467092416426)
 
+    def test_pdf(self):
+        # regression tests for improvements introduces in gh-13665
+        # these tests are bases on Gyner & Smyth (2016)
+
+        # infinite mean corresponds to an inverse-chisquare pdf.
+        assert_allclose(
+            stats.invgauss.pdf([-1, 0, 1, 2, np.inf, np.nan], mu=np.inf),
+            [0., 0., 0.24197072, 0.10984782, 0., np.nan],
+            atol=1e-08
+        )
+        # extreme values have zero density for any finite mean value
+        assert_allclose(stats.invgauss.pdf([-1, 0, np.inf], mu=1), [0, 0, 0])
+
+    def test_ppf_isf(self):
+        p = [0.000001, 0.00001, 0.0001, 0.001, 0.01, 0.1, 0.5, 0.9, 0.99,
+             0.999, 0.9999, 0.99999, 0.999999]
+        p1 = stats.invgauss.cdf(stats.invgauss.ppf(p, mu=1), mu=1)
+        abs_error = np.abs(p - p1)
+        # five-number summary of the absolute error
+        fns = np.percentile(
+            abs_error, [0, 25, 50, np.mean(abs_error), 75, 100]
+        )
+        assert_allclose(fns, [0., 0., 1.27054940e-21, 1.96413583e-17,
+                        2.16840434e-19, 2.22044605e-16], atol=1e-12)
+
+        q = stats.invgauss.ppf(p, mu=1)
+        q1 = stats.invgauss.ppf(stats.invgauss.cdf(q, mu=1), mu=1)
+        rel_error = np.abs(q1 - q) / q
+        fns2 = np.percentile(
+            rel_error, [0, 25, 50, np.mean(rel_error), 75, 100]
+        )
+        assert_allclose(fns2, [0., 0., 0., 9.15882970e-17, 1.16804249e-16,
+                        4.92818217e-16], atol=1e-12)
+
+        # tests if algorithm does not diverge for small probabilities.
+        assert np.isclose(stats.invgauss.ppf(0.00013, mu=1, scale=3),
+                          0.15039762631802803, atol=2.22e-16)
+        # test if it returns right tail values accurately
+        assert np.isclose(stats.invgauss.isf(1e-20, mu=1.5, scale=1 / 0.7),
+                          126.3493, atol=2.22e-16)
```

Comment:
I think there is a different meaning of scale between the paper and SciPy, because these aren't passing, right? SciPy's distribution infrastructure handles `scale` for all distributions. There is no need to test with scales other than 1. If you try a scale other than 1 and it doesn't match what is in a paper, it's more likely that the parameterization is different.